diff --git a/api/__pycache__/main.cpython-310.pyc b/api/__pycache__/main.cpython-310.pyc
index f72bf2d..b5a670b 100644
Binary files a/api/__pycache__/main.cpython-310.pyc and b/api/__pycache__/main.cpython-310.pyc differ
diff --git a/api/main.py b/api/main.py
index adaa930..e0df198 100644
--- a/api/main.py
+++ b/api/main.py
@@ -6,7 +6,7 @@ import wandb
 import pandas as pd
 import sys
 
-from source.mlops.api.preprocessing import *
+from api.preprocessing import *
 
 app = FastAPI()
 
diff --git a/api/train.py b/api/train.py
index 1c93cc5..49acc64 100644
--- a/api/train.py
+++ b/api/train.py
@@ -2,7 +2,7 @@ from sklearn.tree import DecisionTreeClassifier
 from sklearn.model_selection import train_test_split
 from sklearn.preprocessing import LabelEncoder
 
-from source.mlops.modules.preprocessing import *
+from preprocessing import *
 
 
 class Target_encoder(BaseEstimator, TransformerMixin):
