diff --git a/api/__pycache__/__init__.cpython-310.pyc b/api/__pycache__/__init__.cpython-310.pyc
index d8a7da1..160e716 100644
Binary files a/api/__pycache__/__init__.cpython-310.pyc and b/api/__pycache__/__init__.cpython-310.pyc differ
diff --git a/api/__pycache__/main.cpython-310.pyc b/api/__pycache__/main.cpython-310.pyc
index f72bf2d..b5a670b 100644
Binary files a/api/__pycache__/main.cpython-310.pyc and b/api/__pycache__/main.cpython-310.pyc differ
diff --git a/api/__pycache__/preprocessing.cpython-310.pyc b/api/__pycache__/preprocessing.cpython-310.pyc
index a48fb44..2bfe7cf 100644
Binary files a/api/__pycache__/preprocessing.cpython-310.pyc and b/api/__pycache__/preprocessing.cpython-310.pyc differ
diff --git a/api/__pycache__/train.cpython-310.pyc b/api/__pycache__/train.cpython-310.pyc
deleted file mode 100644
index 1e0d34c..0000000
Binary files a/api/__pycache__/train.cpython-310.pyc and /dev/null differ
diff --git a/api/main.py b/api/main.py
index adaa930..e0df198 100644
--- a/api/main.py
+++ b/api/main.py
@@ -6,7 +6,7 @@ import wandb
 import pandas as pd
 import sys
 
-from source.mlops.api.preprocessing import *
+from api.preprocessing import *
 
 app = FastAPI()
 
diff --git a/api/train.py b/api/train.py
index 1c93cc5..603129b 100644
--- a/api/train.py
+++ b/api/train.py
@@ -1,8 +1,17 @@
-from sklearn.tree import DecisionTreeClassifier
-from sklearn.model_selection import train_test_split
-from sklearn.preprocessing import LabelEncoder
-
-from source.mlops.modules.preprocessing import *
+import logging
+from multiprocessing import Value
+import pandas as pd
+from sklearn.preprocessing import MinMaxScaler
+from sklearn.preprocessing import StandardScaler
+from sklearn.preprocessing import LabelEncoder, OneHotEncoder
+from sklearn_pandas import DataFrameMapper
+from sklearn.impute import SimpleImputer
+from sklearn.pipeline import Pipeline, FeatureUnion
+from sklearn.preprocessing import FunctionTransformer
+from sklearn.base import BaseEstimator, TransformerMixin
+import numpy as np
+
+from preprocessing import *
 
 
 class Target_encoder(BaseEstimator, TransformerMixin):
